.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: java
 jdk:
-- oraclejdk7
+- oraclejdk8
```

Comment:
We should still test against jdk7 if possible, as many users depend on this. Try changing `oraclejdk8` to `openjdk7` instead.

Reply:
I've already fixed this on `master`, please rebase ⚡️